Source/Glitch64/OGLglitchmain.cpp

```diff
@@ -29,6 +29,21 @@
  */
 #include <Settings/Settings.h>
 
+#if !defined(_WIN32) && !defined(__ANDROID__) && !defined(ANDROID)
+/*
+ * Do not include <SDL.h> for Windows or Android builds.
+ * Most other platforms tend to include it by default, though.
+ *
+ * Just a few dependencies to SDL remain in the code:
+ *     1.  SDL_SetGammaRamp(), in CorrectGamma()
+ *     2.  SDL_SetGammaRamp(), in grGetGammaTableExt()
+ *     3.  SDL_Surface* m_pScreen = SDL_SetVideoMode(...), to create context
+ * Context creation can be done fine without SDL, and eventually it will be
+ * nice to remove it.  However, Mac & modern Linux tends to come with SDL.
+ */
+#include <SDL.h>

```

Comment:
You have remaining dependencies to SDL in your fork of Glitch64 at:

* [here](https://github.com/project64/project64/blob/0d3d42c7f9cd335f32279d71560d9983054d23f2/Source/Glitch64/OGLglitchmain.cpp#L2803),
* [here](https://github.com/project64/project64/blob/0d3d42c7f9cd335f32279d71560d9983054d23f2/Source/Glitch64/OGLglitchmain.cpp#L2838),
* and ... [here](https://github.com/project64/project64/blob/e5c6fa245bdb5a67c9192d18e7f45acce72de700/Source/Glitch64/OGLglitchmain.cpp#L1041-L1109) (which later you deleted, which just does not work)

Does that answer your question?